frontend/src/hooks/useActions.js

```diff
@@ -199,7 +201,11 @@ const useActions = (registerHotkeys = false) => {
       handler: () => dispatchCustomEvent('sidepanel:open', { panel: 'options' })
     },
     CONVERT_TO_DFA: {
-      // handler: () => console.log('Convert to DFA'),
+      handler: () => {
```

Comment:
```suggestion
      disabled: () => projectType !== 'FSA',
      handler: () => {
```

Reply:
Used it in #349 and remembered I didn't see it here

Reply:
Ah so instead of checking if the project type is FSA I can just add that disabled "flag" instead?

Reply:
ye, and then it will disable itself in the UI